app/src/filedialogex.cpp

```diff
@@ -238,7 +247,6 @@ QString FileDialog::defaultFileName( FileType fileType )
         case FileType::MOVIE: return "untitled.mp4";
         case FileType::SOUND: return "untitled.wav";
         case FileType::PALETTE: return "untitled.xml";
-        default: Q_ASSERT( false );
```

Comment:
Any reason to remove this default tag?

Reply:
Yes because all cases are handled, so having the default would cause a warning, saying it's obsolete.